AsyncDisplayKit/ASVideoNode.mm

```diff
@@ -126,14 +127,16 @@ - (AVPlayerItem *)constructPlayerItem
 {
   ASDN::MutexLocker l(__instanceLock__);
 
-  if (_asset != nil) {
-    AVPlayerItem *playerItem = [[AVPlayerItem alloc] initWithAsset:_asset];
-    playerItem.videoComposition = _videoComposition;
-    playerItem.audioMix = _audioMix;
-    return playerItem;
+  AVPlayerItem *playerItem = nil;
+  if (_assetURL != nil) {
+    playerItem = [[AVPlayerItem alloc] initWithURL:_assetURL];
+  } else if (_asset != nil) {
+    playerItem = [[AVPlayerItem alloc] initWithAsset:_asset];
   }
 
-  return nil;
+  playerItem.videoComposition = _videoComposition;
+  playerItem.audioMix = _audioMix;
+  return playerItem;
```

Comment:
This is what this PR was created for - seems we need to create the playerItem using the URL instead of Asset for streaming-based videos.  Initializing with asset will sometimes cause the video to not load properly.